infra/helm/lumigator/README.md

```diff
@@ -6,10 +6,10 @@ This Helm chart is the official way to deploy Lumigator in cloud environments wi
 
 This Helm chart is composed by two sub-charts:
 
-## `backend` 
+## `backend`
```

Comment:
nit: i don't think we need to change this file (unless this was ruff, then feel free to ignore)